integrated_channels/integrated_channel/transmitters/learner_data.py

```diff
@@ -62,37 +62,46 @@ def single_learner_assessment_grade_transmit(self, exporter, **kwargs):
         )
         kwargs.update(channel_name=app_label)
 
-        # Even though we're transmitting a learner, they can have records per assessment (multiple per course).
-        for learner_data in exporter.single_assessment_level_export(**kwargs):
+        if not self.enterprise_configuration.disable_learner_data_transmissions:
```

Comment:
would it be clearer to use an if block around the log + an early return rather than an if around the whole function?

Reply:
Fair enough!